packages/app-review/android/src/main/java/io/capawesome/capacitorjs/plugins/appreview/AppReview.java

```diff
@@ -1,11 +1,55 @@
 package io.capawesome.capacitorjs.plugins.appreview;
 
+import android.app.Activity;
 import android.util.Log;
+import com.getcapacitor.Logger;
+import com.google.android.gms.tasks.Task;
+import com.google.android.play.core.review.ReviewInfo;
+import com.google.android.play.core.review.ReviewManager;
+import com.google.android.play.core.review.ReviewManagerFactory;
 
 public class AppReview {
 
-    public String echo(String value) {
-        Log.i("Echo", value);
-        return value;
+    private final ReviewManager manager;
+    private final Activity activity;
+
+    public AppReview(Activity activity) {
+        this.activity = activity;
+        this.manager = ReviewManagerFactory.create(activity);
```

Comment:
is it okay to use activity like this? `ReviewManagerFactory.create` expect context

Reply:
Yes, this is okay, but please only call it in the actual implementation method. In this way, the instance is only created when it is actually needed.